config/container_resolvers_conf.xml.sample

```diff
@@ -0,0 +1,35 @@
+<containers_resolvers>
+  <explicit />
+  <!-- explicit: resolves container URI for a job through explict container
+       tags in the tool XML wrapper. -->
+
+  <!-- All mulled flavors below only work if enable_beta_mulled_containers is 
+       set to true in the galaxy.yaml config file.
+
+  <!-- <cached_mulled /> -->
+  <!-- cached_mulled: resolves container URI through bioconda -> mulled
+       automatic mapping, preferring cached containers in the accessible docker
+       engine. Requires docker engine. -->
+
+  <!-- <cached_mulled_singularity /> -->
+  <!-- cached_mulled_singularity: resolves container URI through
+       bioconda -> mulled automatic mapping, preferring cached singularity
+       containers accessible to local singularity binaries. Only works with
+       enable_beta_mulled_containers set to true in the galaxy.yaml config file.
+       -->
+
+  <!-- <mulled /> -->
+  <!-- mulled: resolves container URI through bioconda -> mulled automatic
+       mapping. Doesn't require docker or singularity, and should be the method
+       of choice if mulled resolution is required and the container execution is
+       handled by a container orchestration instead of docker directly. It will
+       also work with direct docker engine tool executions. -->
+
+  <!-- <build_mulled /> -->
+  <!-- build_mulled: builds a docker container locally for one or more bioconda
+       packages listed on the tools definition. -->
+       
+  <!-- <build_mulled_singularity /> -->
+  <!-- build_mulled_singularity: builds a singularity container locally for one
+       or more bioconda packages listed on the tools definition. -->
```

Comment:
`in` or `as part of`.
So maybe `builds a local singularity container for one or more package requirements listed as part of the tools definition.`